src/main/java/org/sonarsource/sonarlint/ls/SonarLintExtendedLanguageServer.java

```diff
@@ -250,4 +250,33 @@ public void setBaseServerUrl(String baseServerUrl) {
 
   @JsonRequest("sonarlint/getServerPathForTokenGeneration")
   CompletableFuture<GetServerPathForTokenGenerationResponse> getServerPathForTokenGeneration(GetServerPathForTokenGenerationParams params);
+
+  class OpenHotspotParams {
```

Comment:
As we have 2 close features, open hotspot in IDE and open hotspot in browser, I would rename this class to disambiguate